lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -1514,14 +1514,11 @@ galaxy:
   #enable_oidc: false
 
   # Sets the path to OIDC configuration file.
-  # The value of this option will be resolved with respect to
-  # <config_dir>.
-  #oidc_config_file: oidc_config.xml
+  oidc_config_file: config/oidc_config.xml
```

Comment:
1) Both oidc config properties are resolved w.r.t. `config_dir` automatically, so the original values are correct.
2) As per @nsoranzo , you should not be modifying this file directly. Any changes should be made to `webapps/galaxy/config_schema.yml`, after which you generate it with `make config-rebuild`.